platform/android/display_server_android.cpp

```diff
@@ -73,6 +73,8 @@ bool DisplayServerAndroid::has_feature(Feature p_feature) const {
 		//case FEATURE_NATIVE_DIALOG:
 		//case FEATURE_NATIVE_DIALOG_INPUT:
 		//case FEATURE_NATIVE_DIALOG_FILE:
```

Comment:
Duplicate case, should remove either this one, or uncomment this one and remove the one below.